zhaquirks/linkind/motion.py

```diff
@@ -0,0 +1,78 @@
+"""Linkind Motion Sensors."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Identify,
+    Ota,
+    PollControl,
+    PowerConfiguration,
+)
+from zigpy.zcl.clusters.homeautomation import Diagnostic
+from zigpy.zcl.clusters.security import IasZone
+
+from zhaquirks import MotionWithReset
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.linkind import LinkindBasicCluster
+
+LINKIND_CLUSTER_ID = 0xFC81
+
+
+class MotionClusterLinkind(MotionWithReset):
+    """Motion cluster."""
+
+    reset_s: int = 60
+
```

Comment:
I have no opinion about that, but some users may want to have 15 seconds and others 120.

Reply:
Total agree, but when not setting this value (leaving motion reset at default), the device was jumpy (Real Motion -> Motion detected -> no motion -> Motion detected) Eve though there was only one Real Motion at the beginning.
With setting this to 60, there were no more issues. 

If you want me to investigate further, I could find out the minimal value neccassary… (tomorrow)

Reply:
Do we know how often the sensor sends the "occupied" status when constantly activated?
For example, the older Xiaomi sensors sends it every 60 seconds and I think zha-quirks uses a 70 second reset delay (should probably be reduced to 60 or just a bit more than 60).

If possible, the lowest delay (plus maybe a few extra seconds) should be chosen.